orientdb/README.md

```diff
@@ -42,16 +42,19 @@ See the next section for the list of configuration parameters for OrientDB.
 ## OrientDB Configuration Parameters
 
 * ```orientdb.url``` - (required) The address to your database.
-    * Supported storage types: memory, plocal
+    * Supported storage types: memory, plocal, remote
     * EX. ```plocal:/path/to/database```
 * ```orientdb.user``` - The user to connect to the database with.
     * Default: ```admin```
 * ```orientdb.password``` - The password to connect to the database with.
     * Default: ```admin```
-* ```orientdb.newdb``` - Create the database if it does not exists, or overwrite the database if it does exist.
+* ```orientdb.newdb``` - Overwrite the database if it already exists.
     * Default: ```false```
+* ```orientdb.remote.storagetype``` - Storage type of the database on remote server
```

Comment:
Yes, a remote connection can be used to connect to a long living plocal or memory database on the server. Memory databases stored on the server don't die like local memory databases do. This being the case you could have two databases with the same name but different types, and so when doing any administration of remote databases you have to know the storage type.